.github/workflows/sync-labels.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     name: Sync labels
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+      - uses: actions/checkout@v3
```

Comment:
This has automatically changed from a SHA to a short semver.
I think this is not expected behaviour!

Not immediately sure why this is happening.